libs/ui/packages/native/storybook/stories/Icon/icons.stories.tsx

```diff
@@ -87,55 +87,10 @@ IconGridStory.args = {
 
 IconGridStory.argTypes = {
   size: {
+    options: ["XS", "S", "M", "L", "XL"],
     control: {
       type: "select",
-      options: ["XS", "S", "M", "L", "XL"],
     },
   },
   color: { control: "color" },
 };
-
-IconGridStory.parameters = {
-  docs: {
-    description: {
-      component: `
-### Adding New Icons to @ledgerHQ/icons-ui Package
-
-1. **Access Figma Core File:**
-   - Navigate to the [Figma Core Library](https://www.figma.com/file/lbUZsVWtBdZpuHSv6lTY9U/%E2%9A%9B%EF%B8%8F-Core-Library?type=design&node-id=12801-12882&mode=design&t=jx0X8jco5kW0wHSP-0).
-
-2. **Export Medium-sized Icons as SVG:**
-   - In Figma, open the medium-sized icons.
-   - Select all the icons you want to export.
-   - Right-click and choose **Export**.
-   - Ensure that the export parameters are already set as needed (no modifications required).
-   - Export the selected icons as SVG files.
-
-3. **Drop Exported SVGs to icons Package:**
-   - Move all the exported SVG files to the \`libs/ui/packages/icons/src/svg\` directory in your project.
-
-4. **Build @ledgerHQ/icons-ui Package:**
-   - Run the command to build the \`@ledgerHQ/icons-ui\` package:
-    \`\`\`bash
-     cd libs/ui/packages/icons && pnpm build
-    \`\`\`
-   - Alternatively, you can build app dependencies using:
```

Comment:
Why do we remove the documentation?

Reply:
Sorry missed to include the .mdx file in the commit (the same as in the react sb pretty much)
Just added it + fixed a TS error.